src/components.yaml

```diff
@@ -1,15 +1,13 @@
-# TODO: Remove NewAnimalSighting / NewUser, they don't fit schemas
-# TODO: Add responses and requestBodies categories for reusable components
 components:
   schemas:
     AnimalSighting:
       type: object
       properties:
-        _id:
-          type: string
-        userId:
-          type: string
-          example: 62a23958e5a9e9b88f853a67
+        # _id:
```

Comment:
Some difficulties with the validation types: string clashing with MongoDB ObjectId's

Reply:
I'm keen to chat about this more. Is it as simple as casting to a string type before sending the ObjectId in the response?